SPECS-SIGNED/grub2-efi-binary-signed/grub2-efi-binary-signed.spec

```diff
@@ -41,6 +41,7 @@ specifically created for installing on %{buildarch} systems
 %package -n     grub2-efi-binary
 Summary:        GRand Unified Bootloader
 Group:          Applications/System
+Requires:       %{name}-mkconfig
```

Comment:
I think %{name} here resolves to "grub2-efi-binary-signed-%{buildarch}", and I think you're looking for %{name}="grub2"